chinese-to-utf8.pl

```diff
@@ -69,7 +69,8 @@
 			}
 		}
 	foreach $l (@default_langs) {
-		if (-r "$m/lang/$l") {
+		# do convert if readable and not a symlink
```

Comment:
checking for symlink should prevent overwriting of an already converted iso-8859-1 lang